### PR TITLE
Fix internal link in setup instructions

### PR DESCRIPTION
**Pull request by fnattino**
For some reason using the `setup.md` file  as target in the link does not seem to work, but referencing `index.md` does the job.

### CODE REVIEW

File: learners/setup.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ command in your terminal (use the **Anaconda prompt** on **Windows**).
    conda install -c conda-forge mamba
    ```
 
-   IMPORTANT: If your terminal responds to the above command with `conda: command not found` see the > <<troubleshooting>> section.
+   IMPORTANT: If your terminal responds to the above command with `conda: command not found` see the [Troubleshooting section](index.md#troubleshooting-conda-command-not-found).
```
Comment (tobyhodges) [inline learners/setup.md:120]:
```suggestion
   IMPORTANT: If your terminal responds to the above command with `conda: command not found` see the [Troubleshooting section](#troubleshooting-conda-command-not-found).
```

The anchor tag should be sufficient to create the internal link here.

**Comment by tobyhodges** [comment]
Within `setup.md` as elsewhere, you can link to a section in the same file with `[link text](#section-id)`. From other pages in the lesson, links written as `[link text](learners/setup.md#section-id)` should resolve to the correct location in the build index.html page of the learner view.

**Comment by fnattino** [comment]
Thanks @tobyhodges for suggesting a better way to link a section within the same file, this solves this specific issue! 

However the suggestion for links within files does not seem to work for me, as the section anchor seems to be lost in the translation: `[link text](learners/setup.md#section-id` becomes `http://127.0.0.1:4321/index.html#setup` on my local deployment.

**Comment by tobyhodges** [comment]
You are absolutely right and that was my mistake, sorry. I will open an issue over at https://github.com/carpentries/workbench to suggest implementing this.

**Comment by fnattino** [comment]
No worries and thanks again for the suggestion. I have opened an issue [here](https://github.com/carpentries/sandpaper-docs/issues/164), will close it in favour of the issue on the workbench repo then!

**Pull request merged**